### PR TITLE
upgrade dependencies

### PR DESCRIPTION
**Pull request by jon-rtr**
Upgrade all dependencies except the following, which require more effort:
```
Package            Current Wanted Latest Package Type URL
sequelize          4.44.3  4.44.3 5.19.6 dependencies https://sequelize.org/
```

https://stackoverflow.com/questions/54603955/node-express-js-unhandled-rejection-error-err-http-headers-sent

### CODE REVIEW

**Comment by redbmk** [comment]
I haven't used material-ui probably since this example was written, but it looks like material UI is around v4.5 now. Just curious why only upgrade to v3.x...are there pretty big breaking changes between versions 3 and 4 but from 1 to 3 aren't too bad?

**Comment by jon-rtr** [comment]
Hi @redbmk! I'm toying around with upgrading the dependencies of this example to be recent versions, and I've run into some challenges.

You're correct that material-ui is up to 4.y. The issue is that the 4.y series is dependent upon newer versions of react than is pinned in the package.json file and I haven't resolved those upgrade pains, yet.

**Comment by jon-rtr** [comment]
@redbmk I've updated most of the dependencies and have it working locally.

File: README.md [hunk 1]
```diff
@@ -1,67 +1,68 @@
-# Basic CRUD App with Node + React
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-This example app shows how to create a Node.js API and display its data with a React UI. To follow along step-by-step, [check out the blog post](https://developer.okta.com/blog/2018/07/10/build-a-basic-crud-app-with-node-and-react).
+## Available Scripts
 
-This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```
Comment (redbmk) [inline README.md]: @jon-rtr I think this is really the only line of the README that needs to change. Could you please restore the rest of the readme?

File: src/index.js [hunk 2]
```diff
@@ -20,6 +20,10 @@ ReactDOM.render(
   </BrowserRouter>,
   document.getElementById('root'),
 );
-registerServiceWorker();
+
+// If you want your app to work offline and load faster, you can change
+// unregister() to register() below. Note this comes with some pitfalls.
+// Learn more about service workers: https://bit.ly/CRA-PWA
+serviceWorker.register();
```
Comment (redbmk) [inline src/index.js]: I think the default now is to set it to `unregister()`. I think that's probably better for this demo and then if someone wants to register the service worker they can change it locally.

**Comment by redbmk** [comment]
Looks great, thanks!

**Pull request merged**